src/vs/workbench/contrib/tasks/browser/task.contribution.ts

```diff
@@ -553,5 +553,10 @@ configurationRegistry.registerConfiguration({
 			],
 			default: 'always',
 		},
+		[TaskSettingId.VerboseLogging]: {
+			type: 'boolean',
+			description: nls.localize('task.verboseLogging', "Enable verbose logging for tasks."),
+			default: false
+		},
```

Comment:
Can we use trace logging instead of a new setting?

Reply:
no, I investigated that a few iterations ago and ultimately gave up.

Reply:
https://github.com/microsoft/vscode/pull/192195